introduction-commandline.html

```diff
@@ -53,14 +53,15 @@
+<p>This page is an introduction to working with Gregorio from the command line. It is intended for users who are familiar with using <span class="latex">L<span class="alpha">a</span>T<span class="epsilon">e</span>X</span> and compiling <span class="latex">L<span class="alpha">a</span>T<span class="epsilon">e</span>X</span> documents at the command line. If that is not the case for you, you can start with <a href="./introduction-editor.html">an editor</a> for <span class="tex">T<span class="epsilon">e</span>X</span> documents.</p>
 
-<p>The command line tool <code>gregorio</code> converts a score file written in <a href="/gabc/index.html">gabc</a> notation into a TeX file. That TeX file contains commands for the GregorioTeX package.</p>
-<p>Your LaTeX file incorporates the score into a document by means of the GregorioTeX command <code>\includescore</code>.  This command takes one argument, the filename of your gabc score. When invoked, it automatically runs <code>gregorio</code> to compile the gabc into a TeX file if needed, and adds the TeX file to its compilation.  (See UserManual.pdf for more details.)</p>
+<p>The command line tool <code>gregorio</code> converts a score file written in <a href="./gabc/index.html">gabc</a> notation into a TeX file. That <span class="tex">T<span class="epsilon">e</span>X</span> file contains commands for the Gregorio<span class="tex">T<span class="epsilon">e</span>X</span> package.</p>
 
-<p>Gregorio extensively uses the new features of LuaTeX, so the compilation needs to occur with this engine. Also, in order to compile your scores automatically, <code>lualatex</code> needs to be able to run external commands. To allow that, run <code>lualatex</code> with the <code>--shell-escape</code> switch, or add <code>gregorio</code> to the list of commands allowed under restricted command execution: the list is named <code>shell_escape_commands</code> in <code>texmf.cnf</code>.
+<p>Your <span class="latex">L<span class="alpha">a</span>T<span class="epsilon">e</span>X</span> file incorporates the score into a document by means of the Gregorio<span class="tex">T<span class="epsilon">e</span>X</span> command <code>\gregorioscore</code>.  This command takes one or two arguments: The first argument is optional and should be enclosed in square brackets (<code>[]</code>) if it is given.  This argument is a single letter, <code>a</code>, <code>f</code>, or <code>n</code>, and indicates the compilation behavior that should be observed for this score.  The default value of this optional argument can be set with a package option or via a macro specifically designed for this purpose.  See the GregorioRef for more details.  The second argument is required and is the filename of your gabc score (enclosed in curly braces: <code>{}</code>).</p>
```

Comment:
In the browser, the `--` is separated from the `shell-escape` because it is at the end of a line.  I think these hyphens should be changed to non-breaking hyphens (`&#8209;`) to prevent line breaks there.